src/test/java/sirius/kernel/event/TestEventHandler.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.kernel.event;
+
+import sirius.kernel.di.std.Register;
+
+@Register(classes = EventHandler.class)
```

Comment:
@Register reicht, interfaces werden automatisch in classes aufgenommen, nur die class selber net